GameServerLib/Chatbox/Commands/JunglespawnCommand.cs

```diff
@@ -17,9 +18,412 @@ public JunglespawnCommand(ChatCommandManager chatCommandManager, Game game)
         }
 
         public override void Execute(int userId, bool hasReceivedArguments, string arguments = "")
-        {
-            _logger.Warn($"{ChatCommandManager.CommandStarterCharacter}{Command} command not implemented");
-            ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "Command not implemented");
+        {
+            var split = arguments.ToLower().Split(' ');
+            if (split.Length < 2)
+            {
+                ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.SYNTAXERROR);
+                ShowSyntax();
+                return;
+            }
+
+            if (!int.TryParse(split[1], out var number))
+            {
+                return;
+            }
+            switch (number)
+            {
+                case 1: //normal only
+                    BlueTeam01();
+                    BlueTeam02();
+                    RedTeam01();
+                    RedTeam02();
+                    break;
+                case 2: // Epic only
+                    Dragon();
+                    Baron();
+                    break;
+                case 3: // normal + Epic
+                    BlueTeam01();
+                    BlueTeam02();
+                    RedTeam01();
+                    RedTeam02();
+                    Dragon();
+                    Baron();
+                    break;
+                default:
+                    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.SYNTAXERROR);
+                    ShowSyntax();
+                    break;
+            }
+            //_logger.Warn($"{ChatCommandManager.CommandStarterCharacter}{Command} command not implemented");
+            //ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "Command not implemented");
+        }
+
+        private void BlueTeam01()

```

Comment:
These function names don't really look self-explanatory, can we change them with something like `Spawn(Blue|Red)(Left|Right)Jungle` and `Spawn(Dragon|Baron)` please?